### PR TITLE
HTS precompile tests for token default status

### LINKED ISSUE #404: Add htsPrecompile acceptance test support for defaultStatus method
### Problem

No htsPrecompile acceptance test support exists for defaultStatus method

### Solution

Implementations of the following precompile methods:

- `getTokenDefaultFreezeStatus(address)`
- `getTokenDefaultKycStatus(address)`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by georgi-l95**
Signed-off-by: georgi-l95 <glazarov95@gmail.com>

**Description**:
Add tests for HTS precompile `getTokenDefaultFreezeStatus` and `getTokenDefaultKycStatus`.

**Related issue(s)**:

Fixes #404 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 1]
```diff
@@ -361,4 +361,50 @@ describe('HTS Precompile Acceptance Tests', async function () {
       expect(pauseStatusAfter).to.equal(false);
     });
   });
+
+  describe('HTS Precompile Default Status Tests', async function() {
+    it('should be able to get default freeze status for fungible token', async () => {
+      const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+      
+      const txTokenDefaultFreezeStatus = await baseHTSContract.getTokenDefaultFreezeStatusPublic(HTSTokenContractAddress, {gasLimit: 1000000});
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:369]: nit: These 5 lines can be share by both the freeze methods

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 2]
```diff
@@ -361,4 +361,50 @@ describe('HTS Precompile Acceptance Tests', async function () {
...
+      expect(responseCodeStatus).to.equal(TX_SUCCESS_CODE);
+
+      const freezeStatus = (await txTokenDefaultFreezeStatus.wait()).events.filter(e => e.event === 'TokenDefaultFreezeStatus')[0].args.defaultFreezeStatus;
+      expect(freezeStatus).to.equal(false);
+    });
+
+    it('should be able to get default kyc status for fungible token', async () => {
+      const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+      
+      const txTokenDefaultKycStatus = await baseHTSContract.getTokenDefaultKycStatusPublic(HTSTokenContractAddress, {gasLimit: 1000000});
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:391]: nit: These 5 lines can be share by both the kyc methods

**Review by Nana-EC — APPROVED** [review]
LG.
Potential for some code sharing if you circle back

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 1 above)
Comment (georgi-l95) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts]: I've extended the existing freeze tests to implement this default status method.

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 2 above)
Comment (georgi-l95) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts]: I've prepared the kyc test here, starting with this default one. Will continue working on KYC test in issue #407

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**